packages/react/context-menu/src/ContextMenu.tsx

```diff
@@ -182,51 +144,51 @@ const ContextMenuContent = React.forwardRef((props, forwardedRef) => {
   };
 
   return (
-    <InsideContentContext.Provider value={true}>
-      {isSubmenu ? (
-        <MenuPrimitive.Content {...commonProps} ref={forwardedRef} />
+    <ContentContext.Provider value={true}>
+      {context.isRootMenu ? (
+        <ContextMenuRootContent {...commonProps} ref={forwardedRef} />
```

Comment:
Not sure if we've been doing this in others, but the `ref` is also a common prop I guess.

Reply:
We've kept it separate in other cases afaik. Leaving it out of `commonProps` aligns better in my head because of the typical spread pattern we use with regular props e.g. `<Component {...props} ref={forwardedRef}`/>`

Reply:
Yeah I was wondering that too.

Reply:
yeah, we haven't done that elsewhere, and I find it easier to see that we definitely passed refs through correctly when they're in the render like this.

Reply:
🚀